db/patch.js

```diff
@@ -4,4 +4,4 @@
 
 // The expected patch level of the database. Update if you add a new
 // patch in the schema/ directory.
-module.exports.level = 7
```

Comment:
A stoopid hack-around is to create a file 'patch-007-006.sql' with just mysql comments so it's an no-op. (But then what happens when you really want to create version 7).